lib/tacos/dependent_slices.py

```diff
@@ -250,10 +251,12 @@ def lines_to_paths(lines: Iterable[str]) -> Generator[OSPath]:
 def main() -> int:
     import fileinput
 
+    config = OSPath(".config/tacos-gha/" + sys.argv.pop())
+
```

Comment:
outdated suggestion kept for completeness:

---

`argv.pop` is an in-place modification that I don't think is the intent here. Also, the behavior when given no arguments will be to use `argv[0]` as the config file. An `IndexError` would be more approrpiate in that case:

```suggestion
    from sys import argv
    config = OSPath(".config/tacos-gha") / argv[1]

```